rubygem-kafo_wizards/rubygem-kafo_wizards.spec

```diff
@@ -0,0 +1,76 @@
+%global gem_name kafo_wizards
+
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+Summary: Define and render wizard-like user interfaces
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.0.1
+Release: 1%{?dist}
+Group: Development/Libraries
+License: GPLv3+
+URL: https://github.com/theforeman/kafo_wizards
+Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
+
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(highline)
+
+%if "%{?scl_ruby}" == "ruby193" || (0%{?el6} && 0%{!?scl:1})
```

Comment:
This should be modernised as ruby193's gone, check the other kafo spec files for an example.